client/my-sites/stats/controller.jsx

```diff
@@ -378,5 +378,30 @@ module.exports = {
 				context.store
 			);
 		}
+	},
+
+	activity_log: function( context ) {
+		let siteId = context.params.site_id;
+		const site = getSite( context.store.getState(), siteId );
+		siteId = site ? ( site.ID || 0 ) : 0;
```

Comment:
In any case, fetching the `siteId` in order to fetch the `siteId` (yes, that's exactly what we're doing) feels a little weird for anyone who views the code for the first time. Maybe we'll want to indicate that we also expect a site slug there. That can be done in a subsequent PR, since similar usages are already found in the stats controller.